server/repository/src/db_diesel/key_value_store.rs

```diff
@@ -185,3 +186,25 @@ impl<'a> KeyValueStoreRepository<'a> {
         Ok(row.and_then(|row| row.value_bool))
     }
 }
+
+pub struct CursorController(KeyValueType);
```

Comment:
This was refactored and applied to places that use cursors, sorry this was done quite early on, I probably should have done after this PR, it adds a little bit of noise to the PR

Reply:
Agree done here: d92564a6175eefdd581de199bcfd2659738a8ca4